clarity/src/vm/mod.rs

```diff
@@ -563,6 +605,25 @@ pub fn execute(program: &str) -> Result<Option<Value>> {
     )
 }
 
+/// Execute for test in Clarity1, Epoch20, testnet.
+#[cfg(any(test, feature = "testing"))]
+pub fn execute_with_max_execution_time(
+    program: &str,
+    max_execution_time: std::time::Duration,
+) -> Result<Option<Value>> {
```

Comment:
I'll admit that this is kind of bikeshedding, but when I read `max_execution_time`, I interpret it as "run for the contract call with the maximum execution limit", so my suggestion is that almost everything that goes by `max_execution_time` should be called some variation of `time_limit`.

Reply:
what about execute_with_limited_execution_time?